examples/README.md

```diff
@@ -41,22 +38,16 @@ It's worth noting, that the `examples` directory is defined as a `npm workspace`
 
 ### Live on production
 
-All code examples are available online. The base URL for the code examples is https://examples.handsontable.com/handsontable/ and after the slash comes the path to the built project. 
+All code examples are available online. The base URL for the code examples is `https://examples.handsontable.com/examples/` and after the slash comes the path to the built project. 
```

Comment:
There is a problem with https://examples.handsontable.com/examples/ that it is not CORS-enabled.

Because of that, the PR https://github.com/handsontable/handsontable/pull/9170 uses https://handsontable.github.io/handsontable instead of https://examples.handsontable.com/examples/. 

Can you add information here that currently both URLs are equivalent and only the GitHub one is CORS-enabled?

Reply:
Added in https://github.com/handsontable/handsontable/pull/9165/commits/5a486fa53cea65673c24677f8e069adb5002063a#diff-49aaa2819e35a856818ecec8c9fa7e1c79ad028d3f44bd749736353cfb51bac9R51